lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/CommonServiceFactory.java

```diff
@@ -30,56 +30,37 @@
 import java.time.Clock;
 
-public class ServiceFactory {
-    private final IdentityVerificationService identityVerificationService;
-    private final ContraindicationMapper contraindicationMapper;
-    private final DcsCryptographyService dcsCryptographyService;
+public class CommonServiceFactory {
     private final ConfigurationService configurationService;
-    private final FormDataValidator formDataValidator;
     private final ObjectMapper objectMapper;
     private final CloseableHttpClient httpClient;
     private final AuditService auditService;
     private final HttpRetryer httpRetryer;
     private final EventProbe eventProbe;
     private final Region awsRegion = Region.of(System.getenv("AWS_REGION"));
 
-    public ServiceFactory(ObjectMapper objectMapper)
+    public CommonServiceFactory(ObjectMapper objectMapper)
             throws NoSuchAlgorithmException, InvalidKeyException, CertificateException,
                     InvalidKeySpecException, KeyStoreException, IOException, HttpException {
         this.objectMapper = objectMapper;
         this.eventProbe = new EventProbe();
-        this.formDataValidator = new FormDataValidator();
         this.configurationService = createConfigurationService();
-        this.dcsCryptographyService = new DcsCryptographyService(configurationService);
-        this.contraindicationMapper = new ContraIndicatorRemoteMapper(configurationService);
         this.httpClient = generateHttpClient(configurationService);
         this.auditService = createAuditService(this.objectMapper);
         this.httpRetryer = new HttpRetryer(httpClient, eventProbe);
-        this.identityVerificationService = createIdentityVerificationService(this.auditService);
     }
 
+    /** Creates common services used across CRIs/lambdas */
     @ExcludeFromGeneratedCoverageReport
-    ServiceFactory(
-            ObjectMapper objectMapper,
-            EventProbe eventProbe,
-            ConfigurationService configurationService,
-            DcsCryptographyService dcsCryptographyService,
-            ContraindicationMapper contraindicationMapper,
-            FormDataValidator formDataValidator,
-            CloseableHttpClient httpClient,
-            AuditService auditService,
-            HttpRetryer httpRetryer)
-            throws NoSuchAlgorithmException, InvalidKeyException {
-        this.objectMapper = objectMapper;
-        this.eventProbe = eventProbe;
-        this.configurationService = configurationService;
-        this.dcsCryptographyService = dcsCryptographyService;
-        this.contraindicationMapper = contraindicationMapper;
-        this.formDataValidator = formDataValidator;
+    CommonServiceFactory(CloseableHttpClient httpClient, HttpRetryer httpRetryer)
+            throws NoSuchAlgorithmException, InvalidKeyException, CertificateException,
+                    InvalidKeySpecException {
+        this.objectMapper = new ObjectMapper().registerModule(new JavaTimeModule());
```

Comment:
Should this be a param so were not potentially creating 2 differently configured mappers